lib/__tests__/testLine.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { houseFactory } from "../../tests/factory";
+
+import { config } from "@/lib/config";
+import { sendLineNotify } from "@/lib/sendLineNotify";
+
+test("Send Line Notify", async () => {
+  const randomHouse = houseFactory();
+
+  const resp = await sendLineNotify(
+    `
+  🏠 ${randomHouse.title || ""}
```

Comment:
前面的空白應該都可以清理下（包含這段字串其他部分）

```suggestion
🏠 ${randomHouse.title || ""}
```

Reply:
整理了一下格式，不過看起來好像不支援 markdown 
https://github.com/Yukaii/zuzugo/commit/7991ae2402ed74eb69ce2f6a7f8973fd979f3076
<img width="684" alt="Screenshot 2023-02-12 at 9 31 53 PM" src="https://user-images.githubusercontent.com/11360957/218314057-c8bf9f2f-42b3-4779-b0d1-24b2192787ac.png">

Reply:
的確是不支援，我這陣子測試的東西比較偏向 Line API 的其他方式，至少圖文可以分開送出（貼圖也可以，但無關緊要）。
在測試中有通過了，另外 hashtag 是可以顯示的，可以參考我們的測試 line 群。
Line API Document (messaging API)
目前我只選用 Text msg 與 Image msg
https://developers.line.biz/en/docs/messaging-api/message-types/#text-messages

Reply:
[
<img width="714" alt="截圖 2023-02-13 18 04 55" src="https://user-images.githubusercontent.com/10606663/218428867-756fab8e-2709-4b04-8f7e-1e08f15ff92d.png">
](url)